src/components/content/order/formElements/NumberInput.tsx

```diff
@@ -5,23 +5,40 @@
 
 import { NumberInputEventHandler, DeployParam } from './CommonTypes';
 import { Form, InputNumber } from 'antd';
+import React from 'react';
+import { Rule } from 'rc-field-form/lib/interface';
 
 export function NumberInput({
     item,
     onChangeHandler,
 }: {
     item: DeployParam;
     onChangeHandler: NumberInputEventHandler;
-}): JSX.Element {
+}): React.JSX.Element {
+    const ruleItems: Rule[] = [{ required: item.mandatory }, { type: 'number' }, { min: 0 }];
+    let minimum: number;
+    let maximum: number;
+
+    const setValidator = (_: unknown, value: number) => {
+        if (value <= minimum) {
+            return Promise.reject(new Error(`The new password not match minimum ${minimum}!`));
+        }
+        if (value >= maximum) {
+            return Promise.reject(new Error(`The new password not match maximum ${maximum}!`));
+        }
+        return Promise.reject(new Error('The new password that you entered do not match!'));
+    };
+
+    for (const key in item.valueSchema) {
+        if (key === 'minimum' || key === 'maximum') {

```

Comment:
can you please define a new enum type which has all supported json schema keys.. and then use it here. 
enum must have minimum, maximum, minimumLength, etc.. as allowed values.